misc/compare_pushes.py

```diff
@@ -1,116 +1,70 @@
 #!/usr/bin/env python
-""" Script to compare two pushes from different Treeherder instances"""
+""" Script to compare pushes from a Treeherder instance against production.
```

Comment:
This diff shows as if I'm modifying the old file but in reality it is a new script.